playbooks/byo/openshift-cluster/initialize_groups.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create initial host groups for localhost
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+  - always
+  tasks:
+  - include_vars: cluster_hosts.yml
+  - name: Evaluate group l_oo_all_hosts
+    add_host:
+      name: "{{ item }}"
+      groups: l_oo_all_hosts
```

Comment:
local, something internal to the role that one should never expect to set from the outside